web/ui/react-app/src/pages/graph/Graph.tsx

```diff
@@ -97,12 +90,23 @@ class Graph extends PureComponent<GraphProps, GraphState> {
           selectedExemplarLabels: { exemplar: {}, series: {} },
         },
         () => {
-          this.plot();
+          this.plotSelectedSeriesIndexes();
```

Comment:
Thanks! This branch still doesn't seem to work for me (at the deploy preview with https://deploy-preview-11789--prometheus-react.netlify.app/graph?g0.expr=prometheus_http_requests_total&g0.tab=0&g0.stacked=0&g0.show_exemplars=0&g0.range_input=1h).

No matter if "Show Exemplars" is initially on or off, it always seems to lose the selected series after toggling it. It looks like this is because the toggling of the "Show Exemplars" button always triggers a query reload, which resets the `data` prop of the graph component, which in `componentDidUpdate()` above then resets the `this.selectedSeriesIndexes`.

The right thing to do would probably be:

* When turning off exemplars, don't run another query (and thus don't reset selected series), just don't show exemplars in the graph anymore.
* When turning on exemplars, do run another query. Maybe in this case it's ok to lose selected series. We could also try to keep them, but when running a query anew, there's always the possibility of getting back a completely different set of series, which is why we usually reset the selected series after running a query.